app/src/main/java/io/appium/uiautomator2/model/UiElementSnapshot.java

```diff
@@ -167,6 +168,21 @@ private static void putAttribute(Map<Attribute, Object> attribs, Attribute key,
                 } else {
                     return null;
                 }
+            case EXTRAS:
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
+                    StringBuilder extras = new StringBuilder();
```

Comment:
I'd rather add all extras into a list and then join it to a string using `;` as separator: https://www.tabnine.com/code/java/methods/android.text.TextUtils/join

Reply:
oh, nice. I wondered String.join, but it was available only in newer api level